### DISCUSSION TITLE
Loading Pascal VOC dataset for object detection

### DISCUSSION THREAD

**Discussion by Telephos** [autogluon/autogluon#2733] [Q&A]
This is a dumb question, but I'm trying to load my training dataset that's in Pascal VOC format and I'm having issues. Most examples show a zip file being downloaded and extracted. I already have the dataset in a google drive folder. This is the folder structure:

![image](https://user-images.githubusercontent.com/22378231/213525753-ffdda4e5-6a58-4fb4-9905-eb9c664d8aee.png)

I tried loading it using from_voc() but it runs continuously and never loads the data. 

I change the directory into my the 'VOC2012_train_val' folder and run the function like so...

![image](https://user-images.githubusercontent.com/22378231/213526068-b8e498e5-0114-4910-8098-45045efdc7d5.png)

There isn't a lot of documentation on this function. Should I be using VOCDetection() instead?

CUDA version 11.2
Ubuntu 20.04 (Google Colab)
Autogluon 0.6.2

Thank you for any help.

**Comment by Telephos** [comment]
Follow up. Trying to import the dataset using the VOCDetection method, but this also just seems to run continuously.

![image](https://user-images.githubusercontent.com/22378231/213556272-b256052b-45f7-42f3-b696-fc7cae71c813.png)

I would assume even a dataset around 17k images should take less than half an hour.

**Comment by ghost** [comment]
Or convert it to COCO format, that's what I've done with my custom datasets that were in VOC format.
